onnxruntime/test/perftest/ort_test_session.cc

```diff
@@ -304,6 +344,15 @@ OnnxRuntimeTestSession::OnnxRuntimeTestSession(Ort::Env& env, std::random_device
         } else {
           ov_options[key] = value;
         }
+      } else if (key == "export_ep_ctx_blob") {
```

Comment:
can we use the existing session options
ep.context_enable, ep.context_embed_mode and ep.context_file_path ?